### PR TITLE
Updates RTL Germany services

### PR DESCRIPTION
**Pull request by SmoothMC**
Optimize RTL Germany Service Logos
Rename RTL Service Super RTL to RTL Super in srp and snp (New Name since 07.2023)
Remove unused Super RTL Logos

### CODE REVIEW

**Review by Lexzie — CHANGES_REQUESTED** [review]
I can imagine to prefer a stacked logo, but what is the reason for the colouring?
I checked on the websites and for some the colours are what we have, so what's wrong about that?
RTL: the colours on the website keep changing: https://www.rtl.de/
RTL Crime: https://www.rtl-crime.de/ 
RTL Living: https://www.rtl-living.de/
For RTL Super both old and new logos are different from the website: https://www.rtl-super.de/ or I thing the colours keep changing?

So how do we determine what the right colours are?

File: build-source/logos/rtl.default.svg
Comment (Lexzie) [inline build-source/logos/rtl.default.svg:1]: These colours are indeed correct when checking on their website, https://www.rtl.de/

**Comment by SmoothMC** [comment]
Hey Lexzie,

for now i used the colors from the rtl+ website live tv here
[https://plus.rtl.de/video-tv/live-tv/rtl-1](https://plus.rtl.de/video-tv/live-tv/rtl-1)

But rtl keeps changing the colors for each show or tv-formats like news etc.

**Comment by Lexzie** [comment]
> Hey Lexzie,
> 
> for now i used the colors from the rtl+ website live tv here https://plus.rtl.de/video-tv/live-tv/rtl-1
> 
> But rtl keeps changing the colors for each show or tv-formats like news etc.

That's what I am saying.
Let's keep the colours as they are.
But I am okay with making the logos stacked.

**Comment by SmoothMC** [comment]
Hm ok...I have a look on it.
Even though I like the colors that I picked from rtl+ 😝

**Comment by SmoothMC** [comment]
Hey Lexzie,

i update the colors again. i hope we now ready to merge. ;)

**Review by SmoothMC — COMMENTED** [review]
Update Logo Colors to Website Referenz

**Review by SmoothMC — COMMENTED** [review]
Update Logo Colors to Website Referenz

**Comment by AbuBaniaz** [comment]
@Lexzie 
Is this okay to merge?

**Comment by SmoothMC** [comment]
Yes! ;)

**Comment by SmoothMC** [comment]
@Lexzie do I miss something? Or why we don't merge?

**Comment by TheLegoBatman** [comment]
He most likely doesn't get notified of the tagging of his username.

**Comment by Lexzie** [comment]
> He most likely doesn't get notified of the tagging of his username.

I was, but had too much other things to do.

**Pull request merged**